org.idempiere.zk.iceblue_c.theme/src/web/theme/iceblue_c/css/fragment/login.css.dsp

```diff
@@ -79,10 +79,15 @@
 
 .login-east-panel, .login-west-panel {
 	width: 350px;
-	background-color: #e0f2ff; 
+	background-color: #ffffff; 
 	position: relative;
 	border: none;
 }
+@media screen and (min-width: 700px) {
+	.login-box-header-txt {
```

Comment:
@alanlesc1  - is it intentional?   The "Login" header is not visible anymore.

Reply:
Yes. It was already hidden in mobile screens, and usually is not necessary because the whole screen is a login form. It's quite intuitive to the user